servers/rendering/renderer_rd/renderer_compositor_rd.h

```diff
@@ -39,12 +39,16 @@
 #include "servers/rendering/renderer_rd/renderer_canvas_render_rd.h"
 #include "servers/rendering/renderer_rd/renderer_storage_rd.h"
 #include "servers/rendering/renderer_rd/shaders/blit.glsl.gen.h"
+#include "servers/rendering/renderer_rd/storage_rd/canvas_texture_storage.h"
+#include "servers/rendering/renderer_rd/storage_rd/texture_storage.h"
 #include "servers/rendering/renderer_rd/uniform_set_cache_rd.h"
 
 class RendererCompositorRD : public RendererCompositor {
 protected:
 	UniformSetCacheRD *uniform_set_cache;
 	RendererCanvasRenderRD *canvas;
```

Comment:
I think we should pick between making them singletons or accessing them from here at some point.

Reply:
I'm setting them all up as singletons, if they are access in many places in another class I may store a pointer in a member variable at construction or in a local variable at the start of a method just to condense things a bit, but they're all accessed internally through their get_singleton() method.